customer-commons/src/main/java/no/unit/nva/customer/model/CustomerDb.java

```diff
@@ -32,19 +34,24 @@
     public CustomerDb() {
     }
 
-    private CustomerDb(Builder builder) {
-        setIdentifier(builder.identifier);
-        setCreatedDate(builder.createdDate);
-        setModifiedDate(builder.modifiedDate);
-        setName(builder.name);
-        setDisplayName(builder.displayName);
-        setShortName(builder.shortName);
-        setArchiveName(builder.archiveName);
-        setCname(builder.cname);
-        setInstitutionDns(builder.institutionDns);
-        setFeideOrganizationId(builder.feideOrganizationId);
-        setCristinId(builder.cristinId);
-        setVocabularySettings(builder.vocabularySettings);
+    public static Builder builder() {
+        return new Builder();
+    }
+
+    public static CustomerDb fromCustomerDto(CustomerDto dto) {
```

Comment:
explicit conversion of a DTO to a DAO. Conversion is tested to verify that it contains all fields.

Reply:
Are we to understand CustomerDb as CustomerDao? I guess renaming would make this clearer

Reply:
I support renaming it to CustomerDao

Reply:
I agree, but I want to do it as own PR.